index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script src="js/index.js"></script>
+
+    <div id="container">
+        <nav id="navbar">
+            <header>
+                <h1>It's a technical documentation</h1>
+            </header>
+            <a class="nav-link" href="#Topic_One">Topic One</a>
+            <a class="nav-link" href="#Topic_Two"> Topic Two</a>
+            <a class="nav-link" href="#Topic_Three"> Topic Three</a>
+            <a class="nav-link" href="#Topic_Four"> Topic Four</a>
+            <a class="nav-link" href="#Topic_Five"> Topic Five</a>
+        </nav>
+
+        <main id="main-doc">
+            <section id="Topic_One" class="main-section">
+                <header>
+                    <h1> Topic One </h1>
+                </header>
+                <p>Microverse section</p>
+                <p>Microverse Students are meant to apply before joining the program, so they are paried with some other
+                    applicants all over the world.
+                </p>
+                <code>
+          let firstName = "Stanley";
+          let otherName = "Andres";
+        </code>
+            </section>
+
+            <section id="Topic_Two" class="main-section">
+                <header>
+                    <h1> Topic Two </h1>
+                </header>
+                <p>This is my second meet</p>
+                <p>Microverse is family space for every developer who want to perform in his career, i'm so verry happy
+                    to work with their students... </p>
+                <code>
+              let lastName = "SIMEON"
+        </code>
```

Comment:
Your code need to be write clearly to make it more easy to read by other developer!

Reply:
- [ ] I think your comment should be more polite. So it should also be a suggestion instead of making it look like an order.  For example, you can say, "**_Kindly_** try to make your code more organized and structured code. They will make your project look professional"